app/Observers/CampaignObserver.php

```diff
@@ -233,14 +233,11 @@ protected function saveRpgSystems(Campaign $campaign): void
      */
     protected function saveGenres(Campaign $campaign)
     {
-        if (!request()->has('genres')) {
```

Comment:
Why remove this? Now when I update the campaign by installing a plugin, all the genres get removed :)